instrumentation/armeria-1.0/library/src/main/java/io/opentelemetry/instrumentation/armeria/v1_0/server/ArmeriaServerTracer.java

```diff
@@ -19,7 +19,8 @@
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public class ArmeriaServerTracer
-    extends HttpServerTracer<HttpRequest, RequestLog, ServiceRequestContext, Void> {
+    extends HttpServerTracer<
+        HttpRequest, RequestLog, ServiceRequestContext, ServiceRequestContext> {
```

Comment:
Why this change?

Reply:
hm, I was going to try with `@Nullable Void`, but now can't repro at all, so reverting the change and let's see what CI says..

Reply:
wow, it passed, thx for asking this 👍👍